src/v2/controllers/vreplicaset_controller/proof/helper_lemmas.rs

```diff
@@ -68,4 +69,51 @@ pub proof fn vrs_non_interference_property_equivalent_to_lifted_vrs_non_interfer
     );
 }
 
+// TODO: Prove this lemma.
+// Annoying sequence reasoning.
+#[verifier(external_body)]
+pub proof fn lemma_filtered_pods_set_equals_matching_pods(
```

Comment:
@codyjrivera could you write more descriptive comments for this lemma?

Reply:
Yes -- I can write comments.

Reply:
Is this one already proved or not? Coz I still see the external_body annotation.

Reply:
It's not proven yet -- I had proved the first of the conjuncts but not the other two -- I've outlined roughly what we need to do in the body.

Reply:
OK if feel free to merge this one when you are ready. I've done my review